include/RE/E/ExtraMapMarker.h

```diff
@@ -61,15 +61,24 @@ namespace RE
 		kFalkreathCapitol = 50,
 		kDawnstarCastle = 51,
 		kDawnstarCapitol = 52,
-		kDLC02_TempleOfMiraak = 53,
-		kDLC02_RavenRock = 54,
-		kDLC02_BeastStone = 55,
-		kDLC02_TelMithryn = 56,
-		kDLC02_ToSkyrim = 57,
-		kDLC02_ToSolstheim = 58
+		kDLC02MiraakTemple = 53,
+		kDLC02RavenRock = 54,
+		kDLC02StandingStones = 55,
+		kDLC02TelvanniTower = 56,
+		kDLC02ToSkyrim = 57,
+		kDLC02ToSolstheim = 58,
+		kDLC02CastleKarstaag = 59,
+		kTotalLocationTypes,
+
+		kDoor,
+		kQuestTarget,
+		kQuestTargetDoor,
+		kMultipleQuestTarget,
+		kPlayerSet,
+		kYouAreHere
 	};
 
-	class MapMarkerData
+	class MapMarkerData : public TESFullName
```

Comment:
`MapMarkerData` does not inherit from `TESFullName`

Reply:
I called `TESFullName` functions on it and it worked fine. It looks nicer and is less clunky to use: `marker->GetFullName` vs `marker->locationName.GetFullName()` also I've never seen `TESFullName` being used as standalone member like that?

Reply:
`MapMarkerData` would have a vtable if it inherited from `TESFullName`. This library is primarily about correctness, please revert this change.

Reply:
This change is reverted, marking this conversation as resolved.